pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseData.py

```diff
@@ -349,8 +348,12 @@ def test_get_case_data_download(self, mock_loadCsv, mock_loadGeojson):
         mock_loadCsv.assert_called()
 
         # test case where csv files are incorrect
-        mock_loadCsv.side_effect = [pd.DataFrame(), pd.read_json(
-            os.path.join(directory, "CaseDataNotFull.json"))]
+        mock_loadCsv.side_effect = [pd.DataFrame(),
```

Comment:
Since I added github as another source, there are now three calls to loadCsv which need to be mocked. I did not notice this before since the exception that was raised by mocking only two calls was catched in getCaseData(). When I changed this statement to catch only FileNotFoundErrors, the error here became apparent.